routes/authRoutes.js

```diff
@@ -1,15 +1,23 @@
 const express = require('express');
 const gravatar = require('gravatar');
 const bcrypt = require('bcrypt');
-// const passport = require('passport');
-// const jwt = require('jsonwebtoken');
+const passport = require('passport');
+const jwt = require('jsonwebtoken');
 
 const User = require('../models/User');
+
 // Load input validation
 const validateRegisterInput = require('../utils/validators').register;
+const validateLoginInput = require('../utils/validators').login;
 
 const router = express.Router();
 const saltRounds = 10;
+const payload = {
```

Comment:
This should be inside `then` on line 89. `user` instead of `User`. The very interesting thing is that this actually works this way 😄

Reply:
it didnt want to work with `user` :)